File: ex-14/ex-14_java/Evento.java
public class Evento {
    public static final int Escape = 256;
    public static final int R = 82;

    private boolean fechado;
    private Integer codigoTecla;

    public Evento() {
        this.fechado = false;
        this.codigoTecla = null;
    }

    public boolean isClosed() {
        return fechado;
    }

    public Integer getKeyPressed() {
        return codigoTecla;
    }
}

File: ex-14/ex-14_python/evento.py
class Evento:
    Escape = 256
    R = 82

    def __init__(self):
        self.fechado = False
        self.codigoTecla = None

    def isClosed(self):
        return self.fechado

    def getKeyPressed(self):
        return self.codigoTecla
